src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -887,17 +887,26 @@ public function set_status( $status ) {
 			return false;
 		}
 
+		// Prevent to change the status of a schedule import.
+		if ( $this->post instanceof WP_Post && $this->post->post_status === Tribe__Events__Aggregator__Records::$status->schedule ) {
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 779df65506fa9996981ef62bc94b609d</sub>

Reply:
```php
use Tribe__Events__Aggregator__Records as Records;
```